src/main/java/org/opensearch/sdk/rest/ExtensionRestPathRegistry.java

```diff
@@ -171,9 +189,13 @@ public List<String> getRegisteredDeprecatedPaths() {
      *
      * @param method  the method.
      * @param path  the path.
+     * @param name  the name corresponding to this route.
      * @return A string appending the method and path.
      */
-    public static String restPathToString(Method method, String path) {
+    public static String restPathToString(Method method, String path, Optional<String> name) {
```

Comment:
Originally this was a more important method but I think it's been reduced to a cosmetic/logging/exception-message method as the rest paths are stored in the `pathTrie`.  So I think the plain space-delimited text may not be useful or may be more confusing than using some other custom output here.